terraform/environments/oasys-national-reporting/locals.tf

```diff
@@ -1,17 +1,15 @@
 # define configuration common to all environments here
-# define environment specific configuration in locals_development.tf, locals_test.tf etc.
+# define environment specific configuration in, locals_test.tf etc.
 
 locals {
   baseline_presets_environments_specific = {
-    development   = local.baseline_presets_development
```

Comment:
only remove this when locals_development.tf is removed and the environment is destroyed